src/com/epam/javatrainings/csv/Main.java

```diff
@@ -0,0 +1,18 @@
+package com.epam.javatrainings.csv;
+
+import com.epam.javatrainings.csv.util.CsvWriter;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        //pass IO csv file paths in command line
+//        String csvReadPath = args[0];
+//        String scvWritePath  =  args[1];
+
+        String csvReadPath = "C:\\GEV\\Java Projects\\epam-trainings\\src\\resources\\customers.csv";
```

Comment:
csvInput? 
csvOutput?

Reply:
Dear Sevak , I don't understand your note. What's wrong? My input file is "customers.csv" and output file is "sorted_customers.csv", if you meant it.

Reply:
Dear Sevak , I don't understand your note. What's wrong? My input file is "customers.csv" and output file is "sorted_customers.csv", if you meant it.